### DISCUSSION TITLE
Quick Question, thanks for the hard work

### DISCUSSION THREAD

**Discussion by jpsoutlook** [Eloston/ungoogled-chromium#2249] [Q&A]
How do you roll back to an older version because, I just noticed the most insane thing.  When I click in the searchbox/ominbox I cant see what I am typing nore can I highlight and delete.  I have always preferred....wait for it....control.  I disabled the tree things in settings and when I try again....same.  I have tried to search for a solution but I cant get past this very annoying dealbreaker...not that I will be missed but why I though this browser was for the people.  If we don't have a choice its not for the people?

Almost forgot when I do this all this bs from the past shows up and its very distracting and offputting.

**Comment by networkException** [comment]
It is highly discouraged to use outdated versions as releases usually fix various security issues. This sounds like a bug, please try if upgrading to a newer version solves the issue or if it exists in vanilla chromium (in which case it has nothing to do with us) and open an issue accordingly

**Reply by jpsoutlook** [reply]
Ok thank you so much for the reply.
